assets/css/_notification_drawer.scss

```diff
@@ -57,3 +55,65 @@
     background-color: $color-secondary-medium;
   }
 }
+
+.m-notification-drawer__card--read {
+  background-color: $white;
+  box-shadow: $subtle-box-shadow;
+}
+
+.m-notification-drawer__card--unread {
+  background-color: $color-secondary-light;
+  border: 1px solid $color-secondary-dark;
+  box-shadow: $box-shadow-secondary;
+}
+
+.m-notification-drawer__mark-all-read-link {
+  color: $color-secondary-dark;
+  cursor: pointer;
+  position: absolute;
+  right: 0;
+}
+
+.m-notification-drawer__n-unread {
+  color: $color-component-medium;
+  text-transform: uppercase;
+}
+
+.m-notification-drawer__read-section-header {
+  margin-bottom: 0.5rem;
+  position: relative;
+}
+
+.m-notification-drawer__submenu {
+  background-color: $white;
+  border-radius: 4px;
+  bottom: 1.5rem;
+  box-shadow: 0 2px 4px 0 rgba(126, 126, 126, 0.5);
```

Comment:
Should this color maybe get defined in `_ui_kit.scss`?

Reply:
Naturally, the hardest part of doing this was thinking of an identifier.